### ISSUE TITLE
gamesPlayed

### ISSUE THREAD

**Issue by voblivion** [joshuaferrara/node-csgo#49]
Hi ! Don't know if the project is supposed to be up to date but it seems the launch of CS GO is broken :

```
.CMsgClientGamesPlayed.GamePlayed#games_played is not a Field
```

I fixed the issue changing

```
this._user.gamesPlayed({
    games_played: [{game_id: '730'}]
});
```
to
```
this._user.gamesPlayed([{game_id: '730'}]);
```
in index.js

**Comment by joshuaferrara** [comment]
Are you using node-steam <= 1.1.0? I am not experiencing this issue on my end.

**Comment by joshuaferrara** [comment]
I also just tested against node-steam@1.4.0 and had no issues.

**Comment by joshuaferrara** [comment]
Try latest version on npm (1.5.5, just published) and let me know if you're still having trouble.

**Comment by voblivion** [comment]
I'm using node-steam@1.4.0

Strange. The issue I have is this one : https://github.com/seishun/node-steam/issues/364
I couldn't understand what was the solution.

I then saw this https://github.com/DoctorMcKay/node-steam-user/issues/87 and tried to call gamesPlayed the way they do (which seems more relievant : why encapsulate in a object with a key "games_played" if the function is gamesPlayed ?)

Not sure if it comes from you repo tho. It might come from node-steam-user which rewrite "SteamUser" but I checked the code (https://github.com/seishun/node-steam/blob/master/lib/handlers/user/index.js and https://github.com/DoctorMcKay/node-steam-user/blob/master/components/messages.js) and they do quite the same...

**Comment by joshuaferrara** [comment]
Gotcha. So you're saying the fix you provided above works while using node-steam-user?

I just tested it without using node-steam-user, and had no issues, so if that fix works with and without node-steam-user, I'd like to push it to the repo.

**Comment by voblivion** [comment]
Never tested without node-steam-user but if you did...
Would be nice to contact node-steam-user owner too and see who does "wrong" :)

**Comment by voblivion** [comment]
Actualy I think it would be better to let the user of your package decide when he wants to call "gamesPlayed" and do it himself. But I'm not very aware of how node-steam works so..

**Comment by joshuaferrara** [comment]
`gamesPlayed` is only sent when `launch()` is called, so the user can still determine when they want it to be set.

I've searched around and have seen what other libraries do, and it seems `[{game_id: '730'}]` would be the correct way. This has been updated in ee40a9a and is available in 1.5.6

If this solves your issue, feel free to close it.

**Comment by voblivion** [comment]
Seems better, thx ! (You forgot to do the same thing line 123 ;) )
